client/src/inside/components/mar/med-order.js

```diff
@@ -0,0 +1,63 @@
+
+const SCHEDULED='sched'
+const PRESCRIBED='prn'
+const PRN_KEYS =[]
+for (let i = 1; i <= 6; i++) {
+  PRN_KEYS.push(`prn${i}`)
+}
+
+class ScheduleOptions {
+  static OPTIONS = {
```

Comment:
Add a comment to say:  "This list of keys with schedules must be kept in sync with the schedule dropdown options shown on the Medications page."